crates/rpc/rpc-builder/src/lib.rs

```diff
@@ -1452,15 +1470,20 @@ impl RpcServerConfig {
     /// This consumes the builder and returns a server.
     ///
     /// Note: The server ist not started and does nothing unless polled, See also [RpcServer::start]
-    pub async fn build(mut self) -> Result<RpcServer, RpcError> {
+    pub async fn build(mut self, modules: &TransportRpcModules) -> Result<RpcServer, RpcError> {
```

Comment:
this is a bit awkward now, but I'm okay with this because this should only be called when an instance of TransportRpcModules exists.